persistence/src/main/java/com/mesosphere/usi/storage/PathTrie.java

```diff
@@ -0,0 +1,371 @@
+package com.mesosphere.usi.storage;
```

Comment:
Hm, I'm not sure this is the right place. Should we pull it into a separate pull request as @zen-dog suggested?

Reply:
Removed